### PR TITLE
fix adls handler

### PR DESCRIPTION
**Pull request by CuriousVini**
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Wrangler 4.1.0-SNAPSHOT ............................ SUCCESS [  2.728 s]
[INFO] Wrangler API ....................................... SUCCESS [  5.368 s]
[INFO] Wrangler REST Protocol Classes ..................... SUCCESS [  0.852 s]
[INFO] Wrangler Core ...................................... SUCCESS [ 20.946 s]
[INFO] Wrangler Storage ................................... SUCCESS [  1.863 s]
[INFO] Wrangler Service ................................... SUCCESS [ 40.545 s]
[INFO] Wrangler Testing Framework ......................... SUCCESS [  0.467 s]
[INFO] Wrangler Transform 4.1.0-SNAPSHOT .................. SUCCESS [ 21.791 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:36 min
[INFO] Finished at: 2019-05-25T09:51:43-07:00
[INFO] ------------------------------------------------------------------------

### CODE REVIEW

File: wrangler-service/src/main/java/io/cdap/wrangler/PropertyIds.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
  * This class {@link PropertyIds} is a collection of static strings.
  */
 public final class PropertyIds {
+  // Id of the workspace
```
Comment (nitinmotgi) [inline wrangler-service/src/main/java/io/cdap/wrangler/PropertyIds.java]: None of the other services use this now, so not sure if this is needed. @albertshau

Comment (CuriousVini) [reply wrangler-service/src/main/java/io/cdap/wrangler/PropertyIds.java]: Ah.

**Comment by nitinmotgi** [comment]
LGTM 👍

**Comment by CuriousVini** [comment]
squashing and merging.

**Review by nitinmotgi — APPROVED** [review]
LGTM 👍

**Pull request merged**